backend/controllers/authUserSearch.js

```diff
@@ -16,6 +16,12 @@ const authUserSearchFactory = oauthApi => {
       ? await oauthApi.userSearch(res.locals, { nameFilter })
       : [await oauthApi.getUser(res.locals, { nameFilter })]
 
+    if (!response) {
```

Comment:
will write a test for this in next PR, having problems with the `await` stuff and jest :disappointed: